charts/nextjs/values.yaml

```diff
@@ -30,6 +30,29 @@ nextjs:
   # Additional environment variables from secrets or configmaps
   envFrom: []
 
+# PostgreSQL configuration
+postgresql:
+  # Enable PostgreSQL subchart
+  enabled: true
+  auth:
+    # PostgreSQL username
+    username: nextjs
+    # PostgreSQL database name  
+    database: nextjs
+    # PostgreSQL password will be auto-generated if not specified
+    password: ""
+    # Existing secret containing PostgreSQL password
+    existingSecret: ""
```

Comment:
We need to provide a value for this secret name, this should be the name of the secret that stores DATABASE_URL and should also store the password value it self as well.